faucet/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "miden-node-faucet"
+version = "0.1.0"
+edition = "2021"
```

Comment:
We should do this the same way as we do it in all other crates (e.g., see [here](https://github.com/0xPolygonMiden/miden-node/blob/next/rpc/Cargo.toml)).